roles/openshift_common/vars/infra_vars.yml

```diff
@@ -0,0 +1,5 @@
+role: infra
```

Comment:
Would it work if we move these variables into /defaults/... instead of /vars/... ? Defaults should be easier to override than vars. In this repo vars is not used anywhere, so far defaults did the trick.

Reply:
I created a defaults folder at the openshift_commons role level. Let me know if that works for you.